debezium-testing/debezium-testing-system/src/main/java/io/debezium/testing/system/tools/kafka/builders/KafkaSubscriptionBuilder.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.testing.system.tools.kafka.builders;
+
+import static io.debezium.testing.system.tools.operatorutil.SubscriptionConstants.COMMUNITY_OPERATORS_SOURCE;
+import static io.debezium.testing.system.tools.operatorutil.SubscriptionConstants.INSTALL_PLAN_APPROVAL;
+import static io.debezium.testing.system.tools.operatorutil.SubscriptionConstants.OPENSHIFT_MARKETPLACE_SOURCE_NAMESPACE;
+import static io.debezium.testing.system.tools.operatorutil.SubscriptionConstants.REDHAT_OPERATORS_SOURCE;
+import static io.debezium.testing.system.tools.operatorutil.SubscriptionConstants.SUBSCRIPTION_API_VERSION;
+import static io.debezium.testing.system.tools.operatorutil.SubscriptionConstants.SUBSCRIPTION_KIND;
+
+import io.debezium.testing.system.tools.fabric8.FabricBuilderWrapper;
+import io.fabric8.kubernetes.api.model.ObjectMetaBuilder;
+import io.fabric8.openshift.api.model.operatorhub.v1alpha1.Subscription;
+import io.fabric8.openshift.api.model.operatorhub.v1alpha1.SubscriptionBuilder;
+import io.fabric8.openshift.api.model.operatorhub.v1alpha1.SubscriptionSpecBuilder;
+
+public class KafkaSubscriptionBuilder extends FabricBuilderWrapper<KafkaSubscriptionBuilder, SubscriptionBuilder, Subscription> {
+
+    private static final String STRIMZI_OPERATOR_NAME = "strimzi-kafka-operator";
+    private static final String AMQ_STREAMS_OPERATOR_NAME = "amq-streams";
+
+    protected KafkaSubscriptionBuilder(SubscriptionBuilder builder) {
+        super(builder);
+    }
+
+    public static KafkaSubscriptionBuilder base() {
+        var builder = new SubscriptionBuilder().withKind(SUBSCRIPTION_KIND)
+                .withApiVersion(SUBSCRIPTION_API_VERSION);
+        return new KafkaSubscriptionBuilder(builder);
+    }
+
+    public KafkaSubscriptionBuilder withCommunityConfig() {
+        builder.withMetadata(new ObjectMetaBuilder()
+                .withName(STRIMZI_OPERATOR_NAME)
+                .build())
+                .withSpec(new SubscriptionSpecBuilder()
+                        .withName(STRIMZI_OPERATOR_NAME)
+                        .withSource(COMMUNITY_OPERATORS_SOURCE)
```

Comment:
I would rather go with SubscriptionConstants.COMMUNITY_OPERATORS_SOURCE style and not importing 6 different static variables.